adserver/decisionengine/backends.py

```diff
@@ -367,14 +372,15 @@ def select_flight(self):
                 # to the possible list of flights
                 if any(
                     (
-                        (flight.clicks_needed_today() > 0),
-                        (flight.views_needed_today() > 0),
+                        (flight.clicks_needed_today(self.interval) > 0),
+                        (flight.views_needed_today(self.interval) > 0),
                     )
                 ):
                     # NOTE: takes into account views for CPM ads
                     # Takes eCPM (CTR * CPC for CPC ads) into account
                     weighted_clicks_needed_today = flight.weighted_clicks_needed_today(
```

Comment:
Yea, definitely feels like the code needs a bit of a refactor/rename to take intervals into account.

Reply:
I'm considering renaming all those functions (`*_needed_today`). Thoughts?